src/components/AuthContainer.js

```diff
@@ -13,15 +13,17 @@ import TermsOfService from 'src/pages/TermsOfService'
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute()
   const { isSignedIn, registrationStatus, acceptedTos, profile } = Utils.useStore(authStore)
+  const spinner = () => h(centeredSpinner, { size: 48, style: { right: 'calc(50vw - 24px)', left: 'calc(50vw - 24px)' } })
```

Comment:
The centeredSpinner has a default of `size: 48 `right? Is there a need to have` size: 48` here as well?

Reply:
I want to explicitly set the size to 48 here since the calc is using half the size. If the default size were to change the calculatoin would be a bit off